src/SumoCoders/FrameworkCoreBundle/Form/Type/AbstractImageType.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+class AbstractImageType extends AbstractType
```

Comment:
Whats the reason of making an AbastractImageType and an ImageType if the ImageType doesn't add anything?